src/backend/catalog/system_views.sql

```diff
@@ -871,6 +871,21 @@ CREATE VIEW pg_stat_activity AS
         LEFT JOIN pg_database AS D ON (S.datid = D.oid)
         LEFT JOIN pg_authid AS U ON (S.usesysid = U.oid);
 
+CREATE VIEW pg_stat_session AS
+    SELECT
+            s.pid,
+            s.total_running_time,
+            s.total_running_count,
+            s.total_idle_time,
+            s.total_idle_count,
+            s.total_transaction_idle_time,
+            s.total_transaction_idle_count,
```

Comment:
here should I rename `transaction_idle*` columns to match the actual state names ?
It is `idle_in_transaction`, not `transaction_idle`. Ditto for `_transaction_idle_aborted`